waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/DefaultMetaStoreClientFactory.java

```diff
@@ -87,8 +99,26 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
           // Metastores.
           return Lists.newArrayList(user);
         }
+      case "get_delegation_token":
+        try {
+          base.open(cachedUgi);
+          Object token = doRealCall(method, args, attempt);
+          if (useSasl) {
+            this.delegationToken = (String) token;
+            base.close();
+            String newTokenSignature = base.getHiveConfValue(HiveConf.ConfVars.METASTORE_TOKEN_SIGNATURE.varname,
```

Comment:
why do you have to set the default value "WAGGLEDANCETOKEN" here ?

are you not able to set it in the hive-site.xml and use it here ?

Reply:
can you elaborate a bit more on why do you have to mask a default value for this property ?

Reply:
If you put a null/empty check and only set it when it is actually set in the hive-site.xml, will it not be sufficient ?